### ISSUE TITLE
feature: swap msi afterburner oc profiles

### ISSUE THREAD

**Issue by gustavokei** [billiford/procswap#8]
When gaming you'd want to use your own profile or stock settings since the high memory oc can result in instabilities.

We can already activate an oc profile by adding `"C:\Program Files (x86)\MSI Afterburner\MSIAfterburner.exe" -Profile1` to the miner .bat file, however, I don't think we can apply a gaming profile once mining stops.

**Comment by billiford** [comment]
I'm working on this. I'll have an example up in an hour or two.

**Comment by billiford** [comment]
@gustavokei First off, great find on setting the profile from the command line. That isn't documented very well.

[Release v0.3.0](https://github.com/billiford/procswap/releases/tag/v0.3.0) adds support for this. 

It adds the flag `--priority-script` which takes in a path to a script (batch file) that will run whenever a priority process (e.g. Cyberpunk) has started. The idea here that you will set your Afterburner profile to gaming mode in this batch file.

**Example**

Here's my command (make sure to run as Administrator):
```bash
procswap.exe --priority D:\Steam\steamapps\common --swap C:\Mining\PhoenixMiner\start_miner.bat --priority-script C:\Mining\procswap\priority_script.bat
```
In the above command I set my priorities to be my Steam games directory `D:\Steam\steamapps\common` and my swap process (mining batch file) to `C:\Mining\PhoenixMiner\start_miner.bat`. I've now added a flag `--priority-script` which will run `C:\Mining\procswap\priority_script.bat` whenever a priority `.exe` under the Steam games directory starts.

You will need to set your profile to mining mode in your miner `.bat` file (as you stated).

Here's the contents of `C:\Mining\PhoenixMiner\start_miner.bat`, note that I am setting my Afterburner profile to `Profile1`.
```
REM
REM Example bat file for starting PhoenixMiner.exe to mine ETH
REM

setx GPU_FORCE_64BIT_PTR 0
setx GPU_MAX_HEAP_SIZE 100
setx GPU_USE_SYNC_OBJECTS 1
setx GPU_MAX_ALLOC_PERCENT 100
setx GPU_SINGLE_ALLOC_PERCENT 100

REM Set the profile to be the mining profile.
"C:\Program Files (x86)\MSI Afterburner\MSIAfterburner.exe" -Profile1

REM IMPORTANT: Replace the ETH address with your own ETH wallet address in the -wal option (Rig001 is the name of the rig)
C:\Mining\PhoenixMiner\PhoenixMiner.exe -pool us1.ethermine.org:4444 -pool2 us2.ethermine.org:4444 -wal 8BAFF6bE0ecb72A55eB442ccae5A2f5B96AfFB3e -worker rig01
pause
```
Here's the contents of `C:\Mining\procswap\priority_script.bat`, the script that runs whenever a game starts up. I set my Afterburner profile to my gaming profile here.
```
@echo off
"C:\Program Files (x86)\MSI Afterburner\MSIAfterburner.exe" -Profile2
```
I've tested this locally and it switches between profiles flawlessly, let me know your results.
